ui/app/styles/core/buttons.scss

```diff
@@ -14,9 +14,9 @@
   display: inline-block;
   font-size: $size-8;
   font-weight: $font-weight-semibold;
-  min-height: 2.25rem;
-  min-width: 6rem;
-  padding: 0.5625rem 0.9375rem;
+  min-height: calc($size-6 + $size-5);
```

Comment:
Is there reasoning behind using `calc()` in some places and a non-variable `rem` value in others when a size variable equivalent doesn't exist?

Reply:
I used calc() when I could use the variables to get the rem value I was trying to replace. There was a case or two in which I kept the rem value because there wasn't an equivalent variable or calculation I could do to get to the value.